src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep/help.html

```diff
@@ -120,3 +120,8 @@
     <a href="https://jenkins.io/blog/2019/02/21/credentials-masking/" target="_blank">Limitations of Credentials Masking</a>
     blog post for more background.
 </p>
+<h4>Note</h4>
```

Comment:
This note can just be deleted. It is already covered in https://github.com/jenkinsci/credentials-binding-plugin/blob/9abc42b55ae68d731749388aa4f2adbe4655b26b/src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep/help.html#L28-L42